src/Akeneo/Tool/Component/Batch/Model/StepExecution.php

```diff
@@ -575,12 +584,12 @@ public function incrementProcessedItems(int $increment = 1): void
 
     public function getProcessedItems(): int
     {
-        return $this->trackingData[self::TRACKING_DATA_PROCESSED_ITEMS];
+        return $this->trackingData[self::TRACKING_DATA_PROCESSED_ITEMS] ?? 0;
     }
 
     public function getTrackingData(): array
     {
-        return $this->trackingData;
+        return $this->trackingData ?? self::TRACKING_DATA_DEFAULT;
```

Comment:
https://github.com/akeneo/pim-community-dev/blob/2ae052ae8ff752991ce9362cf42b902fca4d4bea/src/Akeneo/Tool/Component/Batch/Model/StepExecution.php#L88
```suggestion
        return $this->trackingData;
```